channel/boats.go

```diff
@@ -0,0 +1,206 @@
+package channel
+
+import (
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/Hucaru/Valhalla/constant"
+)
+
+func scheduleBoats(server *Server) {
+	roundUpTime := func(t time.Time, roundOn time.Duration) time.Time {
+		t = t.Round(roundOn)
+
+		if time.Since(t) >= 0 {
+			t = t.Add(roundOn)
+		}
+
+		return t
+	}
+
+	wait := func(duration time.Duration) {
+		timer := time.NewTimer(duration)
+		<-timer.C
+		timer.Stop()
+	}
+
+	r := rand.New(rand.NewSource(time.Now().Unix()))
+
+	server.dispatch <- func() {
+		boatsBoarding(server, true)
+	}
+
+	// Boats take off and land every 15 mins from the hour, with boarding starting 5 mins before
+	for {
+		now := time.Now()
+		delay := roundUpTime(now, 15*time.Minute).Sub(now)
```

Comment:
Are we able to make the clock/timer in-game at the station reflect the appropriate times?

Reply:
I'm not sure what does this. I believe it's a clock so I suspect it might be part of character enter channel?